java/compilers/CFG.java

```diff
@@ -211,6 +211,43 @@ boolean predictSetsDisjoint() {
     }
     //</editor-fold>
 
+    HashMap<Tuple<Symbol, Symbol>, Integer> buildLLParseTable() {
```

Comment:
I've modified this to map to Rule rather than Integer, see db807b685e67d6f97f5010d6902f4d91bded6a78.